integrationtests/as-lucene-directory/src/test/java/org/infinispan/test/integration/as/wildfly/model/Member.java

```diff
@@ -27,7 +37,30 @@
    @GeneratedValue
    private Long id;
 
-   @Field(index = Index.YES, store = Store.NO)
+   @Latitude
+   Double latitude;
+
+   @Longitude
+   Double longitude;
```

Comment:
there's actually no need.
Some people prefer the strong encapsulation and "traditional Java" style, like beans. But some others preferring concise and to the point code will use public field accessors.. both are supported, so it's actually a good idea to test for both in our tests.